verifier/src/test/groovy/com/okta/jwt/impl/OktaJWTClaimsVerifierTest.groovy

```diff
@@ -20,12 +20,12 @@ import com.nimbusds.jose.proc.SimpleSecurityContext
 import com.nimbusds.jwt.JWTClaimsSet
 import com.nimbusds.jwt.proc.BadJWTException
 import com.okta.jwt.JoseException
-import com.okta.jwt.JwtVerifier
-import com.okta.jwt.TestSupport
 import org.testng.annotations.Test
 
 import java.security.NoSuchAlgorithmException
 import java.time.Instant
+import static com.okta.jwt.TestSupport.*
```

Comment:
Oddly enough no.  I tried to do that, I think it is related to an issue with a groovy version.  Removing the wildcard works when the test is run via IntelliJ, but fails to compile via the CLI.  We can try to update the groovy version/plugin but that is defined outside of this project.
(but normally i would agree with you for most static imports), my own personal exception is defining hamcrest Matcher imports as importing `*` makes it easier to type complete tests